### ISSUE TITLE
 if you need the label mask of the style image in the testing phase

### ISSUE THREAD

**Issue by justchenhao** [ZPdesu/SEAN#6]
Greeting!
I want to ask a question about if you need the label mask of the style image in the testing phase. I notice that the label mask of the style image and the input layout may be very different, e.g. the samples in the ADE20K dataset. When encoding the style,  it is natural to use a corresponding segmentation mask to do the "region-wise average pooling" for the style image.
Thank you.

**Comment by ZPdesu** [comment]
Yes, you need label mask of the style image in the testing phase but you only need them once. Whenever the styles codes are extracted from the style images, you no longer need the style images and the corresponding label masks any more.
